README.md

```diff
@@ -16,11 +16,16 @@ The polyfill will only be loaded if the URLPattern doesn't already exist on the
 // CJS module loading (Node.js)
 require("urlpattern-polyfill");
 
-// ESM module loading (Node.js)
+// ESM module loading (Node.js and browser)
+if (!globalThis.URLPattern) {
+  await import("urlpattern-polyfill");
+}
+/** 
+ * The above is the recommend way to load the ESM module. 
+ * However, when you can't use top level await,
```

Comment:
or you always want to load it and replace any existing URLPattern object, you can ...

Reply:
The condition is still in the code, and will not overwrite a preexisting implementation.
I will add en export to the url-pattern, so when someone would want to do this, it becomes possible

Reply:
Yeah that is correct, but we should do that and document it

Reply:
exported url-pattern, and updated readme